### DISCUSSION TITLE
[rejected] data wysyłki w offers/listing / allegro polecam

### DISCUSSION THREAD

**Discussion by lisek2000** [allegro/allegro-api#4328] [Suggestions - rejected]
Dzień dobry,

czy do zasobu /offers/listing można dodać datę planowanej wysyłki do kupującego? W przeciwnym wypadku nie ma możliwości rozróżnić ofert gdzie sprzedający posiada produkty od tzw. preorderów.

Czy planują Państwo  w przyszłości udostępnić także zasoby do programu Allegro Polecam?

**Comment by MaciejFrackowiak** [comment]
Twoją sugestię dot. daty planowanej wysyłki w /offers/listing dopisuję na listę sugestii dot. rozwoju REST API.

Natomiast odnośnie Allegro Polecam - to program stricte dla Kupujących, natomiast zasoby REST API rozwijamy głównie dla Sprzedających.

**Comment by lisek2000** [comment]
Dziękuję za wiadomość. Mam nadzieję, że uda się taką funkcjonalność dodać w najbliższym czasie - czekam z niecierpliwością :)

**Comment by AureliuszBrussy** [comment]
Nie planujemy rozszerzać zakresu danych zwracanych w [GET /offers/listing](https://developer.allegro.pl/documentation/#operation/getListing).
